### ISSUE TITLE
ui team section

### ISSUE THREAD

**Issue by DanaEFalah** [facn6/week1ch#29]
i think that it was better to keep the team's details under the team section

**Comment by samjam48** [comment]
Agreed. It seems odd to have a team section and then have actual details about the team member in a different area. it would be cool to click on the person and have your details revealed underneath the photo perhaps.

**Comment by g4ss4n** [comment]
@samjam48 @DanaEFalah Issue fixed 🥚
